spec/spec_helper.rb

```diff
@@ -10,4 +10,4 @@
 
 RSpec.configure do |config|
   config.formatter = :documentation
-end
```

Comment:
We'll definitely want to make sure we're writing tests first for our future methods.  Makes it harder to test a PR to make sure it's good to merge.

Reply:
Definitely got my tests written now, sorry for misunderstanding the assignment! I caught the bit about writing tests after my initial pull request.